make install does not work

~/projects/experiments/laravel-roadrunner-in-docker   master  make install
```docker-compose run --rm --user "1000:1000" --no-deps app composer install --ansi --prefer-dist
Building app
Step 1/12 : FROM --platform=linux/amd64 php:8.0.9-alpine as runtime
 ---> d75050b53d70
Step 2/12 : COPY --from=composer:2.1.5 /usr/bin/composer /usr/bin/composer
 ---> Using cache
 ---> 3448bdd0f8c1
Step 3/12 : COPY --from=spiralscout/roadrunner:2.3.3 /usr/bin/rr /usr/bin/rr
 ---> Using cache
 ---> 3b4dc4e4a3a1
Step 4/12 : ENV COMPOSER_HOME="/tmp/composer"
 ---> Using cache
 ---> 987a6c6a96e5
Step 5/12 : RUN set -x     && apk add --no-cache         postgresql-libs         icu-libs     && apk add --no-cache --virtual .build-deps         postgresql-dev         autoconf         openssl         make         g++     && docker-php-ext-install -j$(nproc)         pdo_pgsql         sockets         opcache         pcntl         intl         1>/dev/null     && pecl install -o redis 1>/dev/null     && echo 'extension=redis.so' > ${PHP_INI_DIR}/conf.d/redis.ini     && wget -q "https://github.com/aptible/supercronic/releases/download/v0.1.12/supercronic-linux-amd64"          -O /usr/bin/supercronic     && chmod +x /usr/bin/supercronic     && mkdir /etc/supercronic     && echo '*/1 * * * * php /app/artisan schedule:run' > /etc/supercronic/laravel     && openssl req -x509 -nodes -days 1095 -newkey rsa:2048         -subj "/C=CA/ST=QC/O=Company, Inc./CN=mydomain.com"         -addext "subjectAltName=DNS:mydomain.com"         -keyout /etc/ssl/private/selfsigned.key         -out /etc/ssl/certs/selfsigned.crt     && chmod 644 /etc/ssl/private/selfsigned.key     && docker-php-source delete     && apk del .build-deps     && rm -R /tmp/pear     && echo -e "\nopcache.enable=1\nopcache.enable_cli=1\nopcache.jit_buffer_size=32M\nopcache.jit=1235\n" >>         ${PHP_INI_DIR}/conf.d/docker-php-ext-opcache.ini     && php -m     && adduser         --disabled-password         --shell "/sbin/nologin"         --home "/nonexistent"         --no-create-home         --uid "10001"         --gecos ""         "appuser"     && mkdir /app /var/run/rr     && chown -R appuser:appuser /app /var/run/rr     && chmod -R 777 /var/run/rr
 ---> Running in 4350d3a9b350
+ apk add --no-cache postgresql-libs icu-libs
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
(1/8) Installing libgcc (10.3.1_git20210424-r2)
(2/8) Installing libstdc++ (10.3.1_git20210424-r2)
(3/8) Installing icu-libs (67.1-r2)
(4/8) Installing gdbm (1.19-r0)
(5/8) Installing libsasl (2.1.27-r12)
(6/8) Installing libldap (2.4.58-r0)
(7/8) Installing libpq (13.3-r0)
(8/8) Installing postgresql-libs (13.3-r0)
Executing busybox-1.33.1-r3.trigger
OK: 48 MiB in 40 packages
+ apk add --no-cache --virtual .build-deps postgresql-dev autoconf openssl make g++
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.14/community/x86_64/APKINDEX.tar.gz
(1/28) Installing libffi (3.3-r2)
```
...
```build/libtool.m4:52: _AC_PROG_LIBTOOL is expanded from...
build/libtool.m4:39: AC_PROG_LIBTOOL is expanded from...
configure.ac:161: the top level
mkdir: can't create directory 'collator/.libs': File exists
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.14/main: No such file or directory
WARNING: Ignoring https://dl-cdn.alpinelinux.org/alpine/v3.14/community: No such file or directory
+ pecl install -o redis
ERROR: Service 'app' failed to build : The command '/bin/sh -c set -x     && apk add --no-cache         postgresql-libs         icu-libs     && apk add --no-cache --virtual .build-deps         postgresql-dev         autoconf         openssl         make         g++     && docker-php-ext-install -j$(nproc)         pdo_pgsql         sockets         opcache         pcntl         intl         1>/dev/null     && pecl install -o redis 1>/dev/null     && echo 'extension=redis.so' > ${PHP_INI_DIR}/conf.d/redis.ini     && wget -q "https://github.com/aptible/supercronic/releases/download/v0.1.12/supercronic-linux-amd64"          -O /usr/bin/supercronic     && chmod +x /usr/bin/supercronic     && mkdir /etc/supercronic     && echo '*/1 * * * * php /app/artisan schedule:run' > /etc/supercronic/laravel     && openssl req -x509 -nodes -days 1095 -newkey rsa:2048         -subj "/C=CA/ST=QC/O=Company, Inc./CN=mydomain.com"         -addext "subjectAltName=DNS:mydomain.com"         -keyout /etc/ssl/private/selfsigned.key         -out /etc/ssl/certs/selfsigned.crt     && chmod 644 /etc/ssl/private/selfsigned.key     && docker-php-source delete     && apk del .build-deps     && rm -R /tmp/pear     && echo -e "\nopcache.enable=1\nopcache.enable_cli=1\nopcache.jit_buffer_size=32M\nopcache.jit=1235\n" >>         ${PHP_INI_DIR}/conf.d/docker-php-ext-opcache.ini     && php -m     && adduser         --disabled-password         --shell "/sbin/nologin"         --home "/nonexistent"         --no-create-home         --uid "10001"         --gecos ""         "appuser"     && mkdir /app /var/run/rr     && chown -R appuser:appuser /app /var/run/rr     && chmod -R 777 /var/run/rr' returned a non-zero code: 1
make: *** [Makefile:17: install] Ошибка 1
```
Docker version 20.10.8, build 3967b7d
docker-compose version 1.29.2, build 5becea4c
Ubuntu 20.04.2 LTS

--- a/Dockerfile
+++ b/Dockerfile
@@ -11,7 +11,7 @@
 #        && yarn install --frozen-lockfile --no-progress --non-interactive \
 #        && NODE_ENV="production" yarn run prod
 
-# Build application runtime, image page: <https://hub.docker.com/_/php>
+# build application runtime, image page: <https://hub.docker.com/_/php>
 FROM --platform=linux/amd64 php:8.0.9-alpine as runtime
 
 # install composer, image page: <https://hub.docker.com/_/composer>

